### PR TITLE
[WIP] Suppress tests output

### PR DESCRIPTION
**Pull request by dmpetrov**
Fixes the outputs issue from #1598

Kudos to @mroutis for pointing to `--logging-clear-handlers` option.

### CODE REVIEW

File: dvc/logger.py [hunk 1]
```diff
@@ -294,7 +294,24 @@ def _visual_center(line, width):
 
 def setup_handlers():
     if not logger.handlers:
-        logger.handlers = [logging.StreamHandler(), logging.StreamHandler()]
+        temp_log_file = 'tmp.log'
+        logger.handlers = [logging.FileHandler(temp_log_file), logging.FileHandler(temp_log_file)]
+
+
+class MockHandlers(object):
```
Comment (shcheklein) [inline dvc/logger.py]: move into tests

**Comment by efiop** [comment]
@dmpetrov Please install pre-commit hooks as noted in our contribution guide, your code is not formatted by black and thus makes CI fail. To format and check manually use:
```
$ black ./
$ flake8 ./
```

File: dvc/logger.py (hunk 1 above)
Comment (dmpetrov) [reply dvc/logger.py]: Good point

**Comment by dmpetrov** [comment]
Resubmitted another PR

**Pull request closed without merging**